folium/__init__.py

```diff
@@ -29,6 +32,14 @@
     raise ImportError('branca version 0.3.0 or higher is required. '
                       'Update branca with e.g. `pip install branca --upgrade`.')
 
+if sys.version_info < (3, 0):
+    warnings.warn(
+        ("folium will stop working with Python 2.7 starting Jan. 1, 2019."
+         " Please transition to Python 3 before this time."
+         " Check out https://python3statement.org/ for more info."),
+        PendingDeprecationWarning
```

Comment:
I never used `PendingDeprecationWarning` but this seems to not print the warning when importing the library. Is that expected?

Reply:
Thanks for the clarification.